src/os/capture/ffmpegencoder.js

```diff
@@ -0,0 +1,353 @@
+goog.provide('os.capture.ffmpegEncoder');
+
+goog.require('os.capture');
+goog.require('os.capture.AbstractVideoEncoder');
+goog.require('os.capture.GifEncoder');
+goog.require('os.capture.WebMEncoder');
+goog.require('os.defines');
+
+
+
+/**
+ * A ffmpeg video encoder.
+ * @extends {os.capture.AbstractVideoEncoder}
```

Comment:
I know we don't have a lint rule for this, but classes/constructors should begin with an uppercase letter to be consistent with the rest of the code.